src/components-examples/journey-maps/angular/journey-maps-full/util/array-utils.ts

```diff
@@ -0,0 +1,3 @@
+export const distinct = <T>(vals: T[]): T[] => {
```

Comment:
IMO `.distinct()` should be part of the JavaScript Array prototype, just like `.sort()` or like https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#distinct-- in Java. My next best idea was to put it into a utility function that can be reused in an attempt to prevent this logic from being implemented multiple times in the same project/repo.